### ISSUE TITLE
'Undefined variable' bug in import dialog

### ISSUE THREAD

**Issue by libeanim** [nextcloud/contacts#1315]
### Describe the bug
I was importing new contacts from a .vcf file. After the import finished it showed "1 faulty contact". I opened the browser console and saw countless type errors, saying `t is undefined`. While I was looking at it, the same error was logged multiple times a second until everything froze and a Firefox dialogue appeared which allowed me to stop the script (see screenshot).
(The contacts were imported successfully though and accessible after I stopped the script and closed the import dialogue)

I then tried to reproduce the error, reimported the contacts but opened the console straight away. The type errors appeared right from the beginning, even before the faulty contact has been processed (it showed 'faulty contact' only after around 50% had been imported)

**To Reproduce**
1. Go to the Contacts page and open the browser console
2. Click on the settings tab at the bottom left and click on 'Import contacts'
3. Select a .vcf file
4. Wait for the errors to appear in the console
5. If everything freezes, wait for Firefox to ask you to stop the script

**Expected behavior**
The import should not cause a crash

**Actual behavior**
The import causes the browser to freeze until the script is stopped.

**Screenshots**
![faulty-contact](https://user-images.githubusercontent.com/1412303/67617823-febb5100-f7de-11e9-85fb-874cdfde181e.png)

![Script crash](https://user-images.githubusercontent.com/1412303/67617991-ecdaad80-f7e0-11e9-8077-a45609f19ad8.png)

#### Browser log
<details>
<summary>How to access your browser console (Click to expand)</summary>

# Chrome
- Press either CTRL + SHIFT + J to open the “console” tab of the Developer Tools.
- Alternative method:
    1. Press either CTRL + SHIFT + I or F12 to open the Developer Tools.
    2. Click the “console” tab.

# Safari
- Press CTRL + ALT + I to open the Web Inspector.
- See Chrome’s step 2. (Chrome and Safari have pretty much identical dev tools.)

# IE9
1. Press F12 to open the developer tools.
2. Click the “console” tab.

# Firefox
- Press CTRL + SHIFT + K to open the Web console (COMMAND + SHIFT + K on Macs).
- or, if Firebug is installed (recommended):
    1. Press F12 to open Firebug.
    2. Click on the “console” tab.

# Opera
1. Press CTRL + SHIFT + I to open Dragonfly.
2. Click on the “console” tab.
</details>

![errors](https://user-images.githubusercontent.com/1412303/67617869-7db08980-f7df-11e9-983b-60dacea5482f.png)

### Server configuration

**Operating system**: Linux

**Web server:** Nginx

**Database:** MariaDB

**PHP version:** 7.3

**Nextcloud version:**  17.0.0.9

**Contacts version:** 3.1.6

**Updated from an older Nextcloud or fresh install:** No

**Signing status:**
```
No errors have been found.
```

**Comment by libeanim** [comment]
I removed the faulty contact and rerun the import, same error messages, also found a slightly different message:
![error-ok-contact](https://user-images.githubusercontent.com/1412303/67618191-fca7c100-f7e3-11e9-9b4c-aca58e88c4ca.png)

**Comment by skjnldsv** [comment]
Hey :)
Duplicate of https://github.com/nextcloud/contacts/issues/942
